src/pages/get-started.astro

```diff
@@ -102,7 +102,7 @@ import Servers from '../components/Servers.vue';
 
   &:after {
     content: '';
-    background: url('/groupincome.org/svgs/bottom-curve-white.svg') no-repeat center bottom;
+    background: url('/svgs/bottom-curve-white.svg') no-repeat center bottom;
```

Comment:
Does this need any special treatment?

Reply:
@taoeffect 
No, the baseURL is not necessary here. it correctly loads in both environments without it.

<img src='https://github.com/okTurtles/groupincome.org/assets/17641213/646e9c81-0fdc-4df0-b940-6f70e887d8bd' width='430'>